indico/web/templates/forms/occurrences_widget.html

```diff
@@ -0,0 +1,159 @@
+{% extends 'forms/base_widget.html' %}
+{% from 'forms/_form.html' import form_field %}
+
+{% block html %}
+    <div class="occurrences-widget i-form-field-fixed-width" id="{{ field.id }}-container">
+        <input type="hidden" name="{{ field.name }}" id="{{ field.id }}" value="{{ field._value() }}" autofocus>
+        <a href="#" class="add-occurrence">Add occurrence</a>
+    </div>
+
+    <script type="text/html" id="{{ field.id }}-template">
+        <div class="occurrence">
+            <input type="text" class="datepicker" readonly>{#--#}
+            <input type="time" class="timepicker" placeholder="HH:MM" pattern="^([0-9]|0[0-9]|1[0-9]|2[0-3]):[0-5][0-9]$">{#--#}
```

Comment:
Maybe we could move the pattern to a macro since we are using it already in several places.